api/v1/tetragon/tetragon.proto

```diff
@@ -45,71 +50,178 @@ message Container {
+    // buffer size is too small to contain the process filename. Consider
+    // increasing buffer size to avoid this.
+    // - `truncArgs` Indicates truncated the processes arguments because the
+    // buffer size was too small to contain all exec args. Consider increasing
+    // buffer size to avoid this.
+    // - `taskWalk` Primarily useful for debugging. Indicates a walked process
+    // hierarchy to find a parent process in the Tetragon buffer. This may
+    // happen when we did not receive an exec event for the immediate parent of
+    // a process. Typically means we are looking at a fork that in turn did
+    // another fork we don't currently track fork events exactly and instead
+    // push an event with the original parent exec data. This flag can provide
+    // this insight into the event if needed.
+    // - `miss` An error flag indicating we could not find parent info in the
+    // Tetragon event buffer. If this is set it should be reported to Tetragon
+    // developers for debugging. Tetragon will do its best to recover
+    // information about the process from available kernel data structures
+    // instead of using cached info in this case. However, args will not be
+    // available.
+    // - `needsAUID` An internal flag for Tetragon to indicate the audit has
+    // not yet been resolved. The BPF hooks look at this flag to determine if
+    // probing the audit system is necessary.
+    // - `errorFilename` An error flag indicating an error happened while
+    // reading the filename. If this is set it should be reported to Tetragon
+    // developers for debugging.
+    // - `errorArgs` An error flag indicating an error happened while reading
+    // the process args. If this is set it should be reported to Tetragon
+    // developers for debugging
+    // - `needsCWD` An internal flag for Tetragon to indicate the current
+    // working directory has not yet been resolved. The Tetragon hooks look at
+    // this flag to determine if probing the CWD is necessary.
+    // - `noCWDSupport` Indicates that CWD is removed from the event because
+    // the buffer size is too small. Consider increasing buffer size to avoid
+    // this.
+    // - `rootCWD` Indicates that CWD is the root directory. This is necessary
+    // to inform readers the CWD is not in the event buffer and is '/' instead.
+    // - `errorCWD` An error flag indicating an error occurred while reading
+    // the CWD of a process. If this is set it should be reported to Tetragon
+    // developers for debugging.
+    // - `clone` Indicates the process issued a clone before exec*. This is the
+    // general flow to exec* a new process, however its possible to replace the
+    // current process with a new process by doing an exec* without a clone. In
+    // this case the flag will be omitted and the same PID will be used by the
+    // kernel for both the old process and the newly exec'd process.
```

Comment:
This field is somehow for debugging, it contains some values that may not change but others may...

Would be better to add something like:
The flags is for debugging purposes only and should not be relied upon to propagate comprehensive information.

Reply:
when you say "add" you mean append this at the end or at the beginning or replacing the whole flag documentation with this?

Reply:
I'm fine adding it at the beginning so it is clear and we keep current doc that you added, the developers are on their own here... ;-)

Reply:
@tixxdz see https://github.com/cilium/tetragon/compare/f4a972a98319ec1f9608f835d04d8524b0aefeda..46156cca63c72f36073b79d8deddfee603b66d69.